packages/cli/src/lib/commands/up/machine.ts

```diff
@@ -101,12 +101,12 @@ export const ensureCustomizedMachine = async ({
       log.debug('Executing machine scripts')
       for (const script of machineDriver.customizationScripts ?? []) {
         // eslint-disable-next-line no-await-in-loop
-        await sshClient.execScript(script)
+        await sshClient.execScript(script, {})
       }
 
-      // ensure docker is accessible
+      log.info('Ensure docker is accessible')
```

Comment:
Not sure this implementation detail should be always printed.

phrasing
```suggestion
      log.info('Ensuring docker is accessible')
```